sunpy/lightcurve/sources/erne.py

```diff
@@ -0,0 +1,56 @@
+from datetime import timedelta,datetime
+from astropy.io import ascii
+from astropy.table import Table, Column
+
+from sunpy.time import TimeRange
+
+def _parse_txt(filepath):
+    """
+    Parses a SOHO/ERNE file from
+    http://srl.utu.fi/erne_data/carrot/carrota.html
+    http://srl.utu.fi/erne_data/carrot/carrotp.html
+
+    """
+    
+    #Reading in Data along with header
+    data = ascii.read(filepath, delimiter = "\s", data_start = 2 ) 
+    header = data.colnames
+    for i in range(len(header)):
+        header[i] = data[0][header[i]]
+    data = data[1:]
+
+    data_modify = []
+    #Converting separate datetime element into a single datetime.datetime column
+    for i in range(len(data)):
+        if int(data['col1'][i]) >= 96:
+            year = 1900 + int(data['col1'][i])
+        else:
+            year = 2000 + int(data['col1'][i])
+       
+        data_modify = data_modify + [TimeRange(datetime(year,int(data['col2'][i]),int(data['col3'][i]),int(data['col4'][i][:2]),int(data['col4'][i][2:])), datetime(year,int(data['col2'][i]),int(data['col3'][i]),int(data['col5'][i][:2]),int(data['col5'][i][2:])))]
```

Comment:
Done with this, doing the same in others as well, also stored the columns in recognizable variables to use at other places.